apps/meteor/app/threads/server/functions.ts

```diff
@@ -21,6 +21,7 @@ export async function reply({ tmid }: { tmid?: string }, message: IMessage, pare
 	];
 
 	await Messages.updateRepliesByThreadId(tmid, addToReplies, ts);
+	await ReadReceipts.incrementThreadMessagesCountById(tmid);
```

Comment:
you don't need to keep track of `tcount` for read receipts.. you just need to flag them as threads

Reply:
I first thought of keeping the counter in order to erase data from the new `message_reads` collection when the counter hits zero... (not in this PR, but we could do that at some point :thinking: )
but I'm not sure if we'd like to erase data from that collection, since once a message turns into a thread, we won't ever turn it into a regular message once again

Reply:
I'll change this to a flag. If we decide to delete data from the `message_reads` collection at some point too, we can use a flag to control that